### PR TITLE
Remove CMSDS

### PR DESCRIPTION
**Pull request by sharmaneil**
## Issue

Can't find the issue this was originally associated with

## About

We have two conflicting style systems: CMS's design system and USWDS. We want to converge to USWDS.

This PR remove's CMS':
- checklists
- buttons
- radio groups

Pending work
- Removing dates (CMS' dates are better and this is work to recreate)
- removing modals
- styling (doesnt matter for MVP too much)
- refactoring the form (not in scope)

## Discussion

The code here *should* work fine, but is worth a second pair of eyes to test. I would be switching over to the apollo integration, but don't want this effort to go to waste in a forgotten branch, so creating a PR for it now

### CODE REVIEW

**Review by timothybest-usds — APPROVED** [review]

**Comment by timothybest-usds** [comment]
:shipit:

File: frontend/src/app/commonComponents/DateInput.jsx
Comment (dmethvin-gov) [inline frontend/src/app/commonComponents/DateInput.jsx:3]: I've been replacing these with the `uniqueIds` that's in my PR. Generating a true UUID is pretty expensive and all we need here is something unique to a particular rendering of the component. If these components are used in snapshot tests we need them to have reproducible id values or they'll generate diffs. If it's easier we can just merge this and update it once my PR lands.

File: frontend/src/app/commonComponents/Dropdown.jsx
Comment (dmethvin-gov) [inline frontend/src/app/commonComponents/Dropdown.jsx:5]: I made some changes to this one on my branch as well, so we'll need to reconcile them. I like the idea of having a default value and "-select-" though, I wanted this too!

File: frontend/src/scss/PrimeStyles.scss
Comment (dmethvin-gov) [inline frontend/src/scss/PrimeStyles.scss:133]: N.B.: USWDS used to have the `horizontal` option but advised only to use it for two choices. Now they are saying always do it vertically. I tend to agree that two options with short names like Yes/No are best done horizontally so no objections here.

File: frontend/src/scss/PrimeStyles.scss
Comment (dmethvin-gov) [inline frontend/src/scss/PrimeStyles.scss:148]: Should we be using the USWDS v2 design tokens consistently? If so I'll need to study up on it. I have had a hard time even understanding the logic behind them, much less use them. For example, why wouldn't the width and max-width above this also use design tokens? If they do, what value would they have?

**Review by dmethvin-gov — APPROVED** [review]
If you want to merge this in first, I can resolve conflicts and update mine.

File: frontend/src/scss/PrimeStyles.scss
Comment (sharmaneil) [reply frontend/src/scss/PrimeStyles.scss:148]: I stole this, along with most of our css and commonComponents, from other USDS projects where designers introduced css. I don't actually know how styles in USWDS works, lol

File: frontend/src/app/commonComponents/DateInput.jsx
Comment (sharmaneil) [reply frontend/src/app/commonComponents/DateInput.jsx:3]: This `DateInput` isn't used atm; it doesn't have on-par validation and error handling compared to the CMSDS one. 

History:
I was originally using https://www.npmjs.com/package/react-html-id, which mostly worked and generated reproducable ids for snapshots. It solved the problem for dynamically generated lists of components, but did not work for GUIDs. For example, `patients-${id}` would conflict with other components rendered at the same time that also coincidentally used `patients-${id}`, since `id` was not a global value and would increment up from 0 in each component. I didn't want the mental overhead of ensuring that prefixes (ie: `patients-`) were globally unique throughout the app, so i switched to `uuid` as a quick fix. 

There is probably some config for this to ensure global unique values, but the library was also designed for class components, not functional components.

Conclusion:
I support stripping this out in favor of your implementation

File: frontend/src/app/commonComponents/Dropdown.jsx
Comment (sharmaneil) [inline frontend/src/app/commonComponents/Dropdown.jsx:21]: not sure if `null` is a valid value for `options`. Might need to be an empty string

**Comment by sharmaneil** [comment]
thanks for the reviews; gonna merge this. Sorry @dmethvin-gov for the conflicts! 👼

**Pull request merged**
